recipes/qt-multimedia/meta.yaml

```
@@ -0,0 +1,113 @@
+{% set version = "6.7.1" %}
+
+package:
+  name: qt6-multimedia
+  version: {{ version }}
+
+source:
+  url: https://download.qt.io/official_releases/qt/{{ version.rpartition('.')[0] }}/{{ version }}/submodules/qtmultimedia-everywhere-src-{{ version }}.tar.xz
+  sha256: 656d1543727f5bf1bd39fe2548ac454860109dc8555df77d7940f21e3d65cd3e
+
+build:
+  number: 0
+  detect_binary_files_with_prefix: true
+  run_exports:
+    - {{ pin_subpackage('qt6-multimedia', max_pin='x.x') }}
+  missing_dso_whitelist:  # [win]
+    - '$RPATH/api-ms-win-core-libraryloader-l1-2-0.dll'  # [win]
+    - '$RPATH/api-ms-win-ntuser-sysparams-l1-1-0.dll'  # [win]
+    - '$RPATH/api-ms-win-core-heap-l2-1-0.dll'  # [win]
+    - '$RPATH/api-ms-win-core-libraryloader-l1-2-1.dll'  # [win]
+
+requirements:
+  build:
+    - make                               # [unix]
+    - cmake
+    - ninja
+    - perl
+    - ruby >=2.5                         # [linux]
+    - bison                              # [linux]
+    - flex                               # [linux]
+    - gperf                              # [linux]
+    - {{ compiler('cxx') }}
+    - {{ compiler('c') }}
+    - {{ stdlib("c") }}
+    - qt6-main {{ version }}             # [build_platform != target_platform]
+    - {{ cdt('mesa-libgl-devel') }}      # [linux]
+    - {{ cdt('mesa-libegl-devel') }}     # [linux]
+    - {{ cdt('mesa-dri-drivers') }}      # [linux]
+    - {{ cdt('libdrm-devel') }}          # [linux]
+    - {{ cdt('libglvnd-glx') }}          # [linux]
+    - {{ cdt('libglvnd-egl') }}          # [linux]
+    - {{ cdt('libglvnd-devel') }}        # [linux]
+    # native libs
+    - ffmpeg                             # [build_platform != target_platform]
+    - gst-plugins-base {{ gstreamer }}   # [build_platform != target_platform and linux]
+    - gstreamer                          # [build_platform != target_platform and linux]
```

```suggestion
    - ffmpeg *=lgpl*                     # [build_platform != target_platform]
```

if upstream is discouraging it we can simply remove it from future versions. 

specifying the lgpl build of ffmpeg will help just solidify that you aren't relying on the GPL stuff keeping the license LPGL